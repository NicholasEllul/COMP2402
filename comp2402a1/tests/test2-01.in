0
This
2
line
4
should
6
print
8

10
space was behind me
12
