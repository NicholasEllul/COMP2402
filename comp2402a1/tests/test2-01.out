0
2
4
6
8
10
12
This
line
should
print

space was behind me
